app/Http/Controllers/API/UserController.php

```diff
@@ -17,7 +17,7 @@ public function show ($username)
 
     public function active($username)
     {
-        $user           = User::where('username', $username)->firstOrFail();
+        $user           = User::where('username', 'LIKE', $username)->firstOrFail();
```

Comment:
Why do you need "LIKE" here? If you dont use space characters as % or _ you should use = so the query is faster.